xbmcvfs.py

```diff
@@ -0,0 +1,176 @@
+# SPDX-License-Identifier: GPL-3.0
+
+import os
+from shutil import copyfile
+
+
+def File(path, flags='r'):  # NOSONAR
+    return open(path, flags, encoding='utf-8')
+
+
+def Stat(path):  # NOSONAR
```

Comment:
Why this weird construction? It should just be a class: 

```python
class State(object):
    def __init__(path):
        """" Initializes a Stat class.

        :param str path:    The file or directory to stat

        """"

        self._stat = os.stat(path)
```

Reply:
I'm not sure, this was like this in the code. I assumed that there needed to be a `Stat()` function or something. Maybe @dagwieers can tell more.